docs/base/readme.md

```diff
@@ -75,26 +73,35 @@ We treat Spec files as an interface, you can construct Spec page in many ways fo
 
 ### Spec Starting Template
 
-<div class="source_note">
-    After initialization, you get `sourcejs/user` folder, which is the place for all your custom content. All new Specs and configuration of main engine must be done there.
-</div>
-
 The starting template for new Spec pages can be found in `sourcejs/docs/starting` folder. Copy the contents to a new folder in `source/user/specs` and you'll be ready to write a new spec.
 
-<a href="/docs/spec" class="source_a_hl">Check the SourceJS Spec page documentation.</a>
+[Check the SourceJS Spec page documentation.](/docs/spec)
 
 ### Server-side Templating Engines
 
-As we mentioned before, it's easy to use other server-side templating engines like Jade, you only need to create a simple SourceJS middleware ([example](https://github.com/sourcejs/sourcejs-jade)) or process your sources with Grunt/Gulp.
+As we mentioned before, it's also possible to use *other* server-side templating engines like Jade. You need to create a SourceJS middleware that will perform the rendering. Our [sourcjejs-jade Middleware](https://github.com/sourcejs/sourcejs-jade) may serve as an example of such middleware.
```

Comment:
Let's keep the name "plugin" for all "sourcejs-*" extensions, middleware is just a type of a plugin, and the name "plugin" in more approachable for non-node devs.